src/content/protocols/pendle.md

```diff
@@ -0,0 +1,417 @@
+---
+protocol: "Pendle"
+website: "https://www.pendle.finance/"
+x: "https://x.com/pendle_fi"
+github: ["https://github.com/pendle-finance"]
+defillama_slug: ["pendle"]
+chain: "Ethereum"
+stage: "O"
+reasons: ["Unverified Contracts"]
+risks: ["L", "H", "L", "H", "H"]
+author: ["CookingCryptos, PaulDeFi"]
+submission_date: "2025-02-19"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+Pendle is a yield-trading protocol that enables users to manage and optimize their yield exposure. By tokenizing yield-bearing assets into standardized yield tokens (SY), Pendle allows these assets to be split into principal tokens (PT) and yield tokens (YT). This separation facilitates various strategies, including earning fixed yields, speculating on future yield changes, and providing liquidity to earn additional rewards. Pendle's Automated Market Maker (AMM) is specifically designed to handle the trading of these time-decaying assets.
+
+# Ratings
+
+## Chain
+
+Pendle is deployed on various chains. This review is based on the Ethereum mainnet deployment of the protocol.
+
+> Chain score: Low
+
+## Upgradeability
+
+The Pendle V2 protocol can be analyzed in a number of logical modules: _Yield Tokenization_, _Pendle AMM Swap_, _vePENDLE, fees and incentives_ and _Governance_. Each module has upgradeable parts. Overall, these vectors could result in the _loss of user funds_, _loss of unclaimed yield_ or otherwise _materially affect the expected performance_ of the protocol. All the control vectors are hold by accounts with insufficient decentralisation and no onchain Governance exists to this date. See the [Security Council Table](#security-council). With the current setup the _Upgradeability Score_ is _High_. 
+
+### Core Protocol Components
+
+Yield Tokenization is the foundation of Pendle, converting yield-bearing assets into standardized yield (SY) tokens which are then split into Principal Tokens (PT) and Yield Tokens (YT). This separation allows users to trade yield exposure independently.
```

Comment:
move this 

> Yield Tokenization is the foundation of Pendle, converting yield-bearing assets into standardized yield (SY) tokens which are then split into Principal Tokens (PT) and Yield Tokens (YT). This separation allows users to trade yield exposure independently.

at the beginning of the _### Yield Tokenization_ Infrastructure section

Delete the _### Core Protocol Components_ section and the two remaining sentences.